codeworld-requirements/codeworld-requirements.cabal

```diff
@@ -0,0 +1,50 @@
+-- Initial requirements-checker.cabal generated by cabal init.  For further
+--  documentation, see http://haskell.org/cabal/users-guide/
+
+name:                codeworld-requirements
+version:             0.1.0.0
+-- synopsis:
+-- description:
+license:             Apache-2.0
+license-file:        LICENSE
+author:              Daniel Marshall
```

Comment:
There are also legal constraints here.  You should identify the "author" as "The CodeWorld Authors", and then add yourself to the AUTHORS and CONTRIBUTORS files in the root dir of the project.  Again, you should be able to follow the lead of any other cabal file in the project.